src/AspNetCoreMulti/Example/Models.cs

```diff
@@ -0,0 +1,17 @@
+namespace Example
+{
+    public class Dog
+    {
+        public string Breed { get; set; }
+    }
+
+    public class Cat
+    {
+        public string Breed { get; set; }
```

Comment:
The first thing I want to ask is why the property is not in the base class? For this example, a one property in the base class and one property in decsendant classes will be useful.

Reply:
Good point. I moved it into an interface to improve clarity.